rootfs/etc/nginx/template/nginx.tmpl

```diff
@@ -739,6 +739,50 @@ stream {
         ssl_stapling                            on;
         ssl_stapling_verify                     on;
         {{ end }}
+
+        {{ if not $all.DisableLua }}
+        ssl_certificate_by_lua_block {
+            local configuration = require("configuration")
```

Comment:
let's extract this logic into a separate Lua module in `rootfs/etc/nginx/lua/` and call that module here, similar to what we do i.e for balancer module.

Reply:
for inspiration you can also look into https://github.com/Shopify/nginx-routing-modules/blob/cdc4070c6893df58d7f59cd522dcaba2a1189434/lua/notary.lua#L22